.github/workflows/unit_tests.yml

```diff
@@ -59,10 +58,14 @@ jobs:
           # NOTE: additional pytest invocations should also add the --cov-append flag
           #       or they will overwrite previous invocations' coverage reports
           #       (for an example, see OVOS Skill Manager's workflow)
+      - name: Maintain Pyee backwards compat
+        run: |
+          pip install -U "pyee>12.0.0"
```

Comment:
above this the test run with 12.1.1 , shouldnt this one be running with the min version we support instead? 8.1.0